lib/home_page.dart

```diff
@@ -192,318 +192,46 @@ class HomeState extends State<Home> with WidgetsBindingObserver {
       child: StreamBuilder<BreezUserModel>(
           stream: widget.userProfileBloc.userStream,
           builder: (context, userSnapshot) {
-            var user = userSnapshot.data;
```

Comment:
I did change those `vars` to `finals` before moving code to make sure it was not being edited at some point, and I keep it because I think it makes senses once that code comes from the streams.